levelupapi/fixtures/games.json

```diff
@@ -1,26 +1,38 @@
 [
-  {
-    "model": "levelupapi.game",
-    "pk": 1,
-    "fields": {
-      "title": "Game1Title",
-      "maker": "Game1Maker",
-      "skill_level": 1,
-      "gamer": 1,
-      "game_type": 1,
-      "number_of_players": 1
+    {
+        "model": "levelupapi.game",
+        "pk": 1,
+        "fields": {
+            "title": "Welcome To",
+            "maker": "Benoit Turpin",
+            "gamer": 1,
+            "game_type": 1,

```

Comment:
The fields being passed in need to match what's in the game model. In the model you have `gametype` but here you have `game_type`